### DISCUSSION TITLE
Pro Tools -> Zoom

### DISCUSSION THREAD

**Discussion by jhelsley** [ExistentialAudio/BlackHole#87] [General]
Tried to follow your examples, but I can't seem to do a simple one mic routing through Pro Tools out to Zoom. Any guidance? 

Thanks so much

**Comment by devinroth** [comment]
Upload a screen shot of how you set it up. I’ll see if I can help.

**Comment by jhelsley** [comment]
Running Mojave on a 2014 Mini, trying to run a mic through a Focusrite Scarlett 18i20 and send the mic and computer audio to Zoom for online demonstration.
![Screen Shot 2020-03-31 at 10 31 57 PM](https://user-images.githubusercontent.com/62973414/78096875-6c783400-73a0-11ea-8564-f79072e0b82b.png)
![Screen Shot 2020-03-31 at 10 32 14 PM](https://user-images.githubusercontent.com/62973414/78096889-700bbb00-73a0-11ea-92a8-cb3fcf7e808c.png)
![Screen Shot 2020-03-31 at 10 32 38 PM](https://user-images.githubusercontent.com/62973414/78096891-70a45180-73a0-11ea-81bf-59b7d9d617cc.png)
![Screen Shot 2020-03-31 at 10 32 56 PM](https://user-images.githubusercontent.com/62973414/78096892-70a45180-73a0-11ea-9be2-420323f82df0.png)
![Screen Shot 2020-03-31 at 10 33 12 PM](https://user-images.githubusercontent.com/62973414/78096893-70a45180-73a0-11ea-86a6-3f4d5d17a217.png)
![Screen Shot 2020-03-31 at 10 34 50 PM](https://user-images.githubusercontent.com/62973414/78096894-713ce800-73a0-11ea-89f3-fbd9fedf3e2f.png)
![Screen Shot 2020-03-31 at 10 35 00 PM](https://user-images.githubusercontent.com/62973414/78096895-713ce800-73a0-11ea-94a4-5e0f2165fcaa.png)

**Comment by devinroth** [comment]
Could you also send a photo of the IO Setup? I suspect that’s where you went wrong.

**Comment by jhelsley** [comment]
![Screen Shot 2020-04-01 at 11 01 00 AM](https://user-images.githubusercontent.com/62973414/78159547-500fe200-7408-11ea-9c2a-55a444164260.png)
![Screen Shot 2020-04-01 at 11 01 08 AM](https://user-images.githubusercontent.com/62973414/78159559-53a36900-7408-11ea-9b86-924d7446fef4.png)

**Comment by devinroth** [comment]
What version of Pro Tools are you using. Try clicking the default button. It might set it up correctly. I only see 18 active channels yet your aggregate shows 34.
